src/Vlingo.Xoom.Streams/Operator/Filter.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+namespace Vlingo.Xoom.Streams.Operator
+{
+  public class Filter<T> : Operator<T, T>
+  {
+    private readonly Predicate<T> _predicate;
+
+    public Filter(Predicate<T> predicate)
+    {
+      _predicate = predicate;
+    }
+
+    public override void PerformInto(T value, Action<T> consumer)
+    {
+      try
+      {
+        if (_predicate.Invoke(value))
+          consumer.Invoke(value);
+      }
+      catch (Exception e)
+      {
+        Streams.Logger.Error($"Filter failed because: {e.Message}", e);
```

Comment:
Will this somehow be handled by Actors logging when used from within an actor? Maybe it was designed in a similar way on Java side.

Reply:
@tjaskula https://github.com/vlingo/xoom-streams/blob/cf914b7aacc71f0927e065c769b8aea402760f76/src/main/java/io/vlingo/xoom/reactivestreams/Streams.java#L61

Reply:
Thanks for clarification.  So we leave it like that for the time being ?

Reply:
I think but if you want to add a log setter to the interface then @hamzajg can also do so in Java -- logger(Logger) not setLogger(Logger).

Reply:
I think it can be done in the later time. I resolve this conversation and waiting @hamzajg to comment on what is left.